web_external/views/widgets/ScoreDetailWidget.js

```diff
@@ -34,6 +45,7 @@ var ScoreDetailWidget = View.extend({
             el.popover({
                 title,
                 container: this.$el,
+                viewport: '.c-submission-score-table',
```

Comment:
It seems to work. See https://github.com/girder/covalic/pull/219/commits/d368b2bde83da530225d0af89d120df470adae36